jobs/release.Jenkinsfile

```diff
@@ -333,6 +332,12 @@ lock(resource: "release-${params.STREAM}", extra: locks) {
                 }
             }
         }
+        stage('Fork Garbage Collection') {
+            build job: 'garbage-collection', wait: false, parameters: [
+                string(name: 'STREAM', value: params.STREAM),
+                booleanParam(name: 'DRY_RUN', value: false)
+            ]
+        }
```

Comment:
Let's hook this up in a separate PR so we can manually run the job to sanity-check it before automating it.